docs/basic_usage.rst

```diff
@@ -136,7 +136,7 @@ Via API
 If you want to control which part of your code should be profiled, you can use
 the `ddtrace.profiling.profiler.Profiler` object::
```

Comment:
Do we want to update this reference too?

Reply:
yes, thank you!